db/setup/schema.sql

```diff
@@ -1,20 +1,19 @@
-CREATE DATABASE ft_transcendence;
+CREATE TABLE
+  IF NOT EXISTS LocalFile(
+    "id" UUID DEFAULT gen_random_uuid() PRIMARY KEY,
+    "filename" VARCHAR(100) NOT NULL UNIQUE,
+    "path" VARCHAR(255) NOT NULL UNIQUE,
+    "mimetype" VARCHAR(255) NOT NULL,
+    "size" BIGINT NOT NULL,
+    "createdAt" TIMESTAMPTZ NOT NULL DEFAULT NOW()
+  );
 
-\c ft_transcendence;
-
-CREATE TABLE IF NOT EXISTS LocalFile(
-  "id" UUID DEFAULT gen_random_uuid() PRIMARY KEY,
-  "filename" VARCHAR(100) NOT NULL UNIQUE,
-  "path" VARCHAR(255) NOT NULL UNIQUE,
-  "mimetype" VARCHAR(255) NOT NULL,
-  "size" BIGINT NOT NULL,
-  "createdAt" TIMESTAMPTZ NOT NULL DEFAULT NOW()
-);
-
-CREATE TABLE IF NOT EXISTS Users(
-  "id" UUID DEFAULT gen_random_uuid() PRIMARY KEY,
-  "username" VARCHAR(20) NOT NULL UNIQUE,
-  "email" VARCHAR(50) NOT NULL UNIQUE,
-  "avatarId" UUID REFERENCES LocalFile(id) UNIQUE,
-  "createdAt" TIMESTAMPTZ NOT NULL DEFAULT NOW()
-);
+CREATE TABLE
+  IF NOT EXISTS Users(
+    "id" UUID DEFAULT gen_random_uuid() PRIMARY KEY,
+    "username" VARCHAR(20) NOT NULL UNIQUE,
+    "email" VARCHAR(50) NOT NULL UNIQUE,
+    "fullName" VARCHAR(255) NOT NULL,
```

Comment:
I formatted this file with [SQL Formatter](https://github.com/sql-formatter-org/sql-formatter) -> `npx sql-formatter db/setup/schema.sql`

The only change is the addition of this field.